SinaiStCatherine/SinaiEt003.xml

```diff
@@ -0,0 +1,428 @@
+                              <locus from="101r" to="110v"/>
+                            </item>
+                            <item xml:id="q13" n="12">
+                              <dim unit="leaf">10</dim>
+                              <locus from="110r" to="120v"/>
+                            </item>
+                            <item xml:id="q14" n="13">
+                              <dim unit="leaf">10</dim>
+                              <locus from="121r" to="129v"/>
+                              <note>The number 122 has been used twice in the foliation, then been corrected to
+                              <locus target="#122"/> and <locus target="#122a"/>.</note>
+                            </item>
+                            <item xml:id="q15" n="14">
+                              <dim unit="leaf">3</dim>
+                              <locus from="130r" to="132v"/>
+                              2, stub after 2
+                            </item>
+                          </list>
+                        </collation>
+                        <condition key="good"/>
+                         </supportDesc>
+
+                     <layoutDesc>
+                       <!--check ruling in ms-->
+                           <layout columns="1" writtenLines="25">
+                             <locus from="3r" to="118r"/>
+                             <dimensions unit="mm" xml:id="textarea1">
+                               <height>135</height>
+                               <width>100</width>
+                             </dimensions>
+                             <ab type="ruling">Ruling and pricking are visible.</ab>
+                             <ab type="ruling" subtype="pattern">1A-1A/0-0/0-0/C</ab>
+                             <ab type="pricking">Horizontal and vertical pricks are visible.</ab>
+                             <ab type="ruling">The upper written line is written above the ruling, the lower written line is written above the ruling.</ab>
+                           </layout>
+                       <layout columns="2" writtenLines="26">
+                         <locus from="118v" to="130v"/>
+                         <dimensions unit="mm" xml:id="textarea2">
+                           <height>135</height>
+                           <width>100</width>
+                         </dimensions>
+                         <ab type="ruling">Ruling and pricking are visible.</ab>
+                         <ab type="ruling" subtype="pattern">1A-1A-1A1A/0-0/0-0/C</ab>
+                         <ab type="pricking">Horizontal and vertical pricks are visible.</ab>
+                         <ab type="ruling">The upper written line is written above the ruling, the lower written line is written above the ruling.</ab>
+                       </layout>
+                     </layoutDesc>
+
+                  </objectDesc>
+                  <handDesc>
+                     <handNote xml:id="h1" script="Ethiopic">
```

Comment:
I do it slightly differently, it would be good to unify at least in this collection.